versions/2.0/vipps-login-migrate-api-1.0-to-2.0.md

```diff
@@ -9,16 +9,18 @@ Reference:\
 ## Table of contents
 * [Overview](#overview)
 * [Convert to vipps login api version 2.0](#convert-to-vipps-login-api-version-20)
+* [Api changes](#api-changes)
 
-## Overview:
+## Overview
 To have a unified way of serving a merchant user information between our applications in Vipps we have implemented a new service handling this responsibility.
 Vipps Login will be the first service to integrate with this new endpoint.
-In vipps Login api version 2.0 we are changing format of scope 'nnin' to 'nin'.
-We have also have a major change to the response from the '/userinfo' endpoint.
-These are changes to the format of claim 'birthdate' from dd.mm.yyyy to yyyy-mm-dd to comply with the ISO 8601 format.
-To comply with the OIDC-standard and to highlight the address that the user has set as default in Vipps the claim 'address' have been converted from list of all addresses to only return the users default address,
-and the none default addresses are served under 'other_address'.
-We will also no longer serve user information in the 'id_token'.
+
+## Api changes
+* Scope 'nnin' is changed to 'nin'
+* Format of claim 'birthdate' is changed from **dd.mm.yyyy** to **yyyy-mm-dd**, to comply with the ISO 8601 format.
+* Claim address have been converted from a list of all address to an object of the default address. All other addresses are served under 'other_address' as a list. 
+This is done to comply with the OIDC-standard which expect address to be a json object.
+* User information is no longer served on the 'id_token'. Therefore, the 'id_token' will only contain user ids.
```

Comment:
Could you add a short description of why we remove information from the id_token?

Reply:
@VidarHolm, added a short description now.